### PR TITLE
add styles file

### PR DESCRIPTION
**Pull request by denver411**
Issue #459 

### Description

Issue #459 first  PR

### Testing steps

Explain how to test your PR manually

*  add attribute data-tooltip='Some message' to any element

### Features PR URL

[PR #ID](https://github.com/howtohireme/give-me-poc-features/pull/#ID)

### Checklist

Make sure that all steps a checked before the merge

- [ ] RSpec tests are passing on CI
- [ ] Cucumber features are passing localy
- [ ] `bin/cop -a` does not return any warnings
- [ ] Tested manually

### Screenshots

Provide screenshots of implemented functionality

### CODE REVIEW

File: app/assets/stylesheets/459-module-for-tooltips-on-any-element.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[data-tooltip] {
```
Comment (Mehonoshin) [inline app/assets/stylesheets/459-module-for-tooltips-on-any-element.css]: и файл хорошо бы переименовать во что-то адекватное

Comment (denver411) [reply app/assets/stylesheets/459-module-for-tooltips-on-any-element.css]: Согласен. Может tooltip.css?

Comment (Mehonoshin) [reply app/assets/stylesheets/459-module-for-tooltips-on-any-element.css]: 👍

Comment (denver411) [reply app/assets/stylesheets/459-module-for-tooltips-on-any-element.css]: data атрибут здесь так добавляется? - data: {tooltid: "some text"}

Comment (Mehonoshin) [reply app/assets/stylesheets/459-module-for-tooltips-on-any-element.css]: да, посмотри доки http://haml.info/

**Pull request merged**